prow/config/config.yaml

```diff
@@ -35,20 +35,16 @@ tide:
   queries:
     - repos:
         - metal3-io/baremetal-operator
-        - metal3-io/base-image
         - metal3-io/cluster-api-provider-metal3
         - metal3-io/ironic-client
         - metal3-io/ironic-hardware-inventory-recorder-image
```

Comment:
how about this repo?

Reply:
I checked it and it has not been archived. I does not really have any other prow config etc, but its not read-only like others I'm removing.